one-ui/src/test/java/com/marklogic/hub/oneui/AbstractOneUiTest.java

```diff
@@ -130,6 +130,16 @@ protected ObjectNode readJsonObject(String json) {
         }
     }
 
+    protected ObjectNode readJsonArrayObject(String json) {
+        try {
+            ObjectNode objectNode = objectMapper.createObjectNode();
+            objectNode.putArray("arrayKey").add(objectMapper.readTree(json));
```

Comment:
It seems you expect an ArrayNode back, right? So just make this "readJsonArray" and have it return an ArrayNode.